pkg/puppetdb/environments_test.go

```diff
@@ -0,0 +1,18 @@
+package puppetdb
+
+import (
+	"testing"
+
+	"github.com/davecgh/go-spew/spew"
+	"github.com/fatih/structs"
+	"github.com/stretchr/testify/require"
+)
+
+// TestFactNames performs a test on the FactNames endpoint and verifies the expected response is returned.
```

Comment:
Looks like a comment from copy/paste, it's only a test file so not major. Could change to:
`// TestEnvironment tests the expected environments are returned`

Reply:
Nice stop - I fixed and pushed an additional commit. I also added "go vet" linting, not that it would have spotted this ...